tests/base.py

```diff
@@ -106,3 +107,11 @@ def tearDown(self) -> None:
         self._conn.close()
         self._process.terminate()
         return super().tearDown()
+
+    @property
+    def gdb_version(self) -> Tuple[int, int]:
+        res = tuple(
+            map(int, re.search(r"(\d+)[^\d]+(\d+)", self._gdb.VERSION).groups())
+        )
+        assert len(res) >= 2
+        return (res[0], res[1])
```

Comment:
No point doing this, we already know it's a tuple of length 2
```suggestion
        return res
```